src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents.mdx

```diff
@@ -100,54 +96,62 @@ Each option for incident preference has its own advantages. You can select any o
 
 By default, a single incident record will be created for each policy.
 
-![A screenshot showing how to change your incident preferences on New Relic One's Alerts user interface.](./images/incident-preference.png "A screenshot showing how to change your incident preferences on New Relic One's Alerts user interface.")
-
-<figcaption>
-  **[one.newrelic.com](https://one.newrelic.com) > Alert & AI > Policies > (select a policy):** The selected policy page shows how alerts rolls up incidents for alert notifications and UI details. (Default is **By policy**). To choose a different option for this policy, click **Incident preference**.
-</figcaption>
-
 To change the incident preference for the selected policy:
 
 1. In the **[one.newrelic.com](https://one.newrelic.com/ "Link opens in a new window.")** top nav, click **Alerts & AI**, then click **Policies**, and then (**select a policy)**.
 2. Click **Incident preference**, select your choice of available [options](#preference-options), and then save.
 
 Repeat these steps for each policy as needed.
 
-## Example: By policy (default) [#preference-policy]
+![A screenshot showing where the incident preference button is inside New Relic One's Alerts user interface.](./images/incident-preference-UI-1.png "A screenshot showing where the incident preference button is inside New Relic One's Alerts user interface.")
```

Comment:
I'd love it if you smudged out my name in this image :D You can use SnagIt to do this.